src/routes/+layout.svelte

```diff
@@ -1,16 +1,3 @@
-<script>
-  import Footer from '../components/Footer.svelte';
-  import JoinSocials from '../components/JoinSocials.svelte';
-</script>
-
-<div class="max-w-page overflow-x-hidden shadow-xl">
-  <div class="h-[100vh] w-[100vw] overflow-x-hidden">
-    <div class="min-h-[calc(100vh-100px)]">
-      <slot />
-      <div class="freq-container mt-[150px]">
-        <JoinSocials />
-      </div>
-    </div>
-    <Footer />
-  </div>
+<div class="max-w-page overflow-x-hidden shadow-xl print:shadow-none">
+  <slot />
```

Comment:
The new default layout is simpler. Also remove the shadow from the print version